app/scripts/controllers/profile-controller.js

```diff
@@ -0,0 +1,58 @@
+/*jshint camelcase: false */
+(function () {
+  'use strict';
+
+  angular
+    .module('porttare.controllers')
+    .controller('ProfileController', ProfileController);
+
+  function ProfileController($auth,
+                            ModalService,
+                            ProfileService,
+                            $ionicLoading,
+                            $ionicPopup,
+                            $scope) {
+    var profileVm = this;
+    profileVm.showNewModal = showNewModal;
+    profileVm.closeModal = closeModal;
+    profileVm.submitProcess = submitProcess;
+    init();
+
+    function init(){
+      ProfileService.getProfile().then(function(res){
+        profileVm.userData = res;
+      });
+    }
+
+    function showNewModal() {
+      profileVm.fecha_de_nacimiento = new Date(profileVm.userData.fecha_de_nacimiento);
```

Comment:
@fabricioflores aqui hago ```new Date```  porque en el formulario cuando al input le digo que sea de tipo date no me reconoce una fecha formateada por moment obtengo un error como este 'Error: [ngModel:datefmt] Expected `2015-05-29T19:06:16.693209Z` to be a date'